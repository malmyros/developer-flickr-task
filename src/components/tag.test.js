import React from 'react';
import { shallow } from 'enzyme';
import Tag from './tag';

describe('tag', () => {
  it('should render without crashing', () => {
    const component = shallow(<Tag title={'title'}/>);
    expect(component).toMatchSnapshot();
  });
});
