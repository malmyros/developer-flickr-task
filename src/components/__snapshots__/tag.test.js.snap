// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag should render without crashing 1`] = `
<span
  className="btn btn-outline-secondary disabled mr-2 mb-2"
>
  title
</span>
`;
